app/api/chapters/chapters.controller.js

```diff
@@ -1,107 +1,66 @@
 const models = require("../../models");
-const logger = require('../../../config/logger');
+const { notFoundError } = require('../../utils/errors');
 
-//get all the chapters
-const getChapters = async (req, res) => {
-  try {
-    const basic_attributes = ["picture_id", "title", "type", "url_media", "description", "zoom", "story"];
-    const longitude = [models.sequelize.literal("ST_X(images.location)"), "longitude"];
-    const latitude = [models.sequelize.literal("ST_Y(location)"), "latitude"];
-    const includeOption = [{
-      model: models.images,
-      attributes: [longitude, latitude],
-    }];
-    const sequelizeQuery = {
-      attributes: basic_attributes,
-      orderBy: ["indexinstory"],
-      include: includeOption
-    };
-    const chapters = await models.stories_chapters.findAll(sequelizeQuery);
-    res.json(chapters);
-  } catch (error) {
-    logger.error(error);
-    res.status(500).json({ error: 'Une erreur s\'est produite lors de la récupération des chapitres.' });
-  }
-};
-
-
-//get a chapter by id
 const getChapterById = async (req, res) => {
   const { id } = req.params;
-  try {
-    const chapter = await models.stories_chapters.findByPk(id);
-    if (chapter) {
-      res.json(JSON.stringify({ "chapters": chapter }));
-    } else {
-      res.status(404).json({ error: 'Aucun chapitre trouvé avec cet ID.' });
-    }
-  } catch (error) {
-    logger.error(error);
-    res.status(500).json({ error: `Une erreur s'est produite lors de la récupération du chapitre avec l'ID ${id}.` });
+  const chapter = await models.stories_chapters.findByPk(id);
+  if (chapter) {
+    res.json(chapter);
+  } else {
+    throw notFoundError(req);
   }
 };
 
-
 //Add a chapter to the db
 const addChapter = async (req, res) => {
-  const { title, type, picture_id, url_media, description, zoom, story, indexinstory } = req.body;
-  try {
-    const newChapter = await models.stories_chapters.create({
+  const { storyId } = req.params;
```

Comment:
owner scope should be checked to make sure only super admin or story owner can add chapter to the story